pkg/system/phase2_creating.go

```diff
@@ -746,6 +746,13 @@ func (r *Reconciler) ReconcileRootSecret() error {
 func (r *Reconciler) ReconcileDB() error {
 	var err error
 	if r.NooBaa.Spec.DBType == "postgres" {
+		// this config map is required by the NooBaaPostgresDB StatefulSet,
+		// if the configMap was not created at this step, NooBaaPostgresDB
+		// would fail to start.
+		if !util.KubeCreateSkipExisting(r.PostgresDBConf) {
```

Comment:
Maybe I missed it, but did you call r.Own(r.PostgresDBConf) ?
You need this to make the configmap be deleted when the noobaa CR is deleted.

Reply:
@guymguym 👍

https://github.com/noobaa/noobaa-operator/compare/5d1098a..42d9cbd

Reply:
Thanks, you don't need to provide commit links on every change, you can just push and update the PR itself.